app/views/guides/new/_new_guide_step_3.html.erb

```diff
@@ -179,8 +178,8 @@
               </ul>
             </div>
             <div class="columns large-9 medium-8">
-              <label id="description" for="action_content" ng-bind="stage.activeAction.description">
-              </label>
+              <!--<label id="description" for="action_content" ng-bind="stage.activeAction.description">
```

Comment:
This is actually something that is put in by the seeds on your local machine. The ones on staging and production don't have any latin.